suite-native/message-system/src/components/MessageSystemRenderer.tsx

```diff
@@ -0,0 +1,46 @@
+import React, { ReactNode } from 'react';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+import { useSelector } from 'react-redux';
+
+import { Box, VStack } from '@suite-native/atoms';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { ContextMessage } from './ContextMessage';
+import { selectAllActiveMessageIds } from '../selectors';
+
+type MessageSystemRendererProps = {
+    children: ReactNode;
+};
+
+const messageSystemRendererStyle = prepareNativeStyle<{ topSafeAreaInset: number }>(
+    (utils, { topSafeAreaInset }) => ({
+        position: 'absolute',
```

Comment:
So in case of non-dismissable banner, the app will be unusable because the bubble will block the navigation. Is it intended?

Reply:
Yes it is intended and aligned with the design.